lib/src/content/components/paginating_group/wolt_modal_sheet_page_transition_state.dart

```diff
@@ -60,17 +60,25 @@ enum WoltModalSheetPageTransitionState {
     }
   }
 
+  /// Returns an animation for the main content slide position based on the
+  /// provided parameters, supporting both LTR and RTL directions.
   Animation<Offset> mainContentSlidePosition(
     AnimationController controller,
     WoltModalSheetPaginationAnimationStyle style, {
     required double sheetWidth,
     required double screenWidth,
     required bool isForwardMove,
+    required bool isLTR, // New parameter to determine the text direction
```

Comment:
```suggestion
    required bool isLTR, // New parameter to determine the text direction
```

Instead of  this, we could use a `textDirection` parameter and pass `Directionality.of(context)`

Reply:
Could we make this parameter optional? That way we can always default to LTR when this parameter is null.

Reply:
Changed the parameter type: 4982dcf65a627145e5f3a59e13dd56aba6e424d4
I would avoid making it optional. First of all, this is not public API. Second, the maintainers should always consider the directionality when calling this.